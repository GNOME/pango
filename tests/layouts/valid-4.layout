{
  "text" : "This paragraph should ac­tual­ly have multiple lines, unlike all the other wannabe äöü pa­ra­graph tests in this ugh test-case. Grow some lines!\n",
  "attributes" : [],
  "font" : "Cantarell 11",
  "width" : 198656
}
