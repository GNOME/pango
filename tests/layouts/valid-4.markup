width=194
This paragraph should ac­tual­ly have multiple lines, unlike all the other wannabe äöü pa­ra­graph tests in this ugh test-case. Grow some lines!
