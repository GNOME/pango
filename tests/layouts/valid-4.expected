This paragraph should ac­tual­ly have multiple lines, unlike all the other wannabe äöü pa­ra­graph tests in this ugh test-case. Grow some lines!

--- parameters

wrapped: 1
ellipsized: 0
lines: 6
width: 194560

--- attributes

range 0 2147483647

--- lines

i=1, index=0, paragraph-start=1, dir=ltr 'This paragraph should ac­tual­'
i=2, index=32, paragraph-start=0, dir=ltr 'ly have multiple lines, unlike '
i=3, index=63, paragraph-start=0, dir=ltr 'all the other wannabe äöü pa­'
i=4, index=96, paragraph-start=0, dir=ltr 'ra­graph tests in this ugh '
i=5, index=124, paragraph-start=0, dir=ltr 'test-case. Grow some lines!
'
i=6, index=152, paragraph-start=1, dir=ltr ''

--- runs

i=1, index=0, chars=30, level=0, gravity=south, flags=4, font=OMITTED, script=latin, language=en-us, 'This paragraph should ac­tual­'
i=2, index=32, no run, line end
i=3, index=32, chars=31, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'ly have multiple lines, unlike '
i=4, index=63, no run, line end
i=5, index=63, chars=29, level=0, gravity=south, flags=4, font=OMITTED, script=latin, language=en-us, 'all the other wannabe äöü pa­'
i=6, index=96, no run, line end
i=7, index=96, chars=27, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'ra­graph tests in this ugh '
i=8, index=124, no run, line end
i=9, index=124, chars=27, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'test-case. Grow some lines!'
i=10, index=151, no run, line end
i=11, index=152, no run, line end
